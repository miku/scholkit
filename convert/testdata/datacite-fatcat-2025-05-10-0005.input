{
  "id": "10.26190/unsworks/26575",
  "type": "dois",
  "attributes": {
    "doi": "10.26190/unsworks/26575",
    "identifiers": [],
    "creators": [
      {
        "name": "Snijder, M",
        "nameType": "Personal",
        "givenName": "M",
        "familyName": "Snijder",
        "nameIdentifiers": [
          {
            "schemeUri": "https://orcid.org",
            "nameIdentifier": "https://orcid.org/0000-0001-8766-3015",
            "nameIdentifierScheme": "ORCID"
          }
        ],
        "affiliation": []
      },
      {
        "name": "Calabria, B",
        "nameType": "Personal",
        "givenName": "B",
        "familyName": "Calabria",
        "nameIdentifiers": [
          {
            "schemeUri": "https://orcid.org",
            "nameIdentifier": "https://orcid.org/0000-0001-6444-4696",
            "nameIdentifierScheme": "ORCID"
          }
        ],
        "affiliation": []
      },
      {
        "name": "Wagemakers, A",
        "nameType": "Personal",
        "givenName": "A",
        "familyName": "Wagemakers",
        "affiliation": [],
        "nameIdentifiers": []
      },
      {
        "name": "Dobbins, T",
        "nameType": "Personal",
        "givenName": "T",
        "familyName": "Dobbins",
        "nameIdentifiers": [
          {
            "schemeUri": "https://orcid.org",
            "nameIdentifier": "https://orcid.org/0000-0003-1841-9056",
            "nameIdentifierScheme": "ORCID"
          }
        ],
        "affiliation": []
      },
      {
        "name": "Shakeshaft, A",
        "nameType": "Personal",
        "givenName": "A",
        "familyName": "Shakeshaft",
        "nameIdentifiers": [
          {
            "schemeUri": "https://orcid.org",
            "nameIdentifier": "https://orcid.org/0000-0002-5472-0930",
            "nameIdentifierScheme": "ORCID"
          }
        ],
        "affiliation": []
      }
    ],
    "titles": [
      {
        "title": "The outcomes of a community-based participatory research project with Aboriginal Australians reducing alcohol-related harms"
      }
    ],
    "publisher": "UNSW Sydney",
    "container": {},
    "publicationYear": 2018,
    "subjects": [],
    "contributors": [
      {
        "name": "UNSW Sydney",
        "nameType": "Organizational",
        "affiliation": [
          {
            "name": "UNSW Sydney",
            "schemeUri": "https://ror.org/",
            "affiliationIdentifier": "https://ror.org/03r8z3t63",
            "affiliationIdentifierScheme": "ROR"
          },
          {
            "name": "UNSW Sydney",
            "schemeUri": "https://www.grid.ac/",
            "affiliationIdentifier": "https://www.grid.ac/grid.1005.4",
            "affiliationIdentifierScheme": "GRID"
          }
        ],
        "contributorType": "HostingInstitution",
        "nameIdentifiers": []
      }
    ],
    "dates": [
      {
        "date": "2018",
        "dateType": "Issued"
      }
    ],
    "language": null,
    "types": {
      "ris": "RPRT",
      "bibtex": "article",
      "citeproc": "article-journal",
      "schemaOrg": "ScholarlyArticle",
      "resourceType": "conference poster",
      "resourceTypeGeneral": "Text"
    },
    "relatedIdentifiers": [],
    "relatedItems": [],
    "sizes": [],
    "formats": [],
    "version": null,
    "rightsList": [
      {
        "rights": "Creative Commons Attribution Non Commercial No Derivatives 4.0 International",
        "rightsUri": "https://creativecommons.org/licenses/by-nc-nd/4.0/legalcode",
        "schemeUri": "https://spdx.org/licenses/",
        "rightsIdentifier": "cc-by-nc-nd-4.0",
        "rightsIdentifierScheme": "SPDX"
      }
    ],
    "descriptions": [
      {
        "description": "39th Annual Meeting and Scientific Sessions of the Society of Behavioral Medicine; New Orleans (LA), United States of America; 2018-04-11 - 2018-04-14",
        "descriptionType": "Other"
      }
    ],
    "geoLocations": [],
    "fundingReferences": [],
    "url": "http://hdl.handle.net/1959.4/unsworks_50188",
    "contentUrl": null,
    "metadataVersion": 2,
    "schemaVersion": "http://datacite.org/schema/kernel-4",
    "source": "mds",
    "isActive": true,
    "state": "findable",
    "reason": null,
    "viewCount": 0,
    "downloadCount": 0,
    "referenceCount": 0,
    "citationCount": 0,
    "partCount": 0,
    "partOfCount": 0,
    "versionCount": 0,
    "versionOfCount": 0,
    "created": "2024-04-15T19:43:30Z",
    "registered": "2024-04-15T19:43:50Z",
    "published": null,
    "updated": "2025-05-10T02:08:49Z"
  },
  "relationships": {
    "client": {
      "data": {
        "id": "unsw.repo",
        "type": "clients"
      }
    }
  }
}
