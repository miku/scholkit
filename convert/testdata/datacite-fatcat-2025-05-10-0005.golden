{
    "source": "datacite",
    "contribs": [
        {
            "extra": {},
            "given_name": "M",
            "raw_name": "M Snijder",
            "role": "author",
            "surname": "Snijder"
        },
        {
            "extra": {},
            "given_name": "B",
            "index": 1,
            "raw_name": "B Calabria",
            "role": "author",
            "surname": "Calabria"
        },
        {
            "extra": {},
            "given_name": "A",
            "index": 2,
            "raw_name": "A Wagemakers",
            "role": "author",
            "surname": "Wagemakers"
        },
        {
            "extra": {},
            "given_name": "T",
            "index": 3,
            "raw_name": "T Dobbins",
            "role": "author",
            "surname": "Dobbins"
        },
        {
            "extra": {},
            "given_name": "A",
            "index": 4,
            "raw_name": "A Shakeshaft",
            "role": "author",
            "surname": "Shakeshaft"
        },
        {
            "extra": {},
            "given_name": "UNSW",
            "index": 5,
            "raw_name": "UNSW Sydney",
            "role": "hostinginstitution",
            "surname": "Sydney"
        }
    ],
    "ext_ids": {
        "doi": "10.26190/unsworks/26575"
    },
    "extra": {
        "arxiv": {},
        "dblp": {},
        "openalex": {
            "open_access": {}
        },
        "crossref": {},
        "oai": {}
    },
    "id": "datacite-8d74f8d4741ebcfeb39695db317f6cc7db697036",
    "publisher": "UNSW Sydney",
    "release_date": "2018-01-01",
    "release_type": "article",
    "release_year": 2018,
    "title": "The outcomes of a community-based participatory research project with Aboriginal Australians reducing alcohol-related harms"
}